client/src/components/ProductList/ProductList.js

```diff
@@ -4,7 +4,30 @@ import styled from 'styled-components/macro'
 import { NavLink } from 'react-router-dom'
 
 export default function ProductList(props) {
-  const { styleId, media, brand, model, retailPrice, sale } = props.product
+  const { styleId, media, brand, model, shops, sale } = props.product
+
+  function shopCount(count) {
+    if (count > 1) {
+      return count + ' Shops'
+    } else {
+      return count + ' Shop'
+    }
+  }
+
+  function minPrice(shops) {
+    const regularPrice = shops
+      .map(product => product.regularPrice)
+      .sort((a, b) => a > b)
+    const salesPrice = shops
+      .map(product => product.salesPrice)
+      .sort((a, b) => a > b)
+    if (salesPrice[0] !== '' && regularPrice[0] > salesPrice[0]) {
+      return salesPrice[0]
+    } else {
+      return regularPrice[0]
+    }
+  }
```

Comment:
```suggestion
    if (salesPrice[0] !== '' && regularPrice[0] > salesPrice[0]) return salesPrice[0]
    return regularPrice[0]
    }
```
This should work as well, and its easier to read ... 

Look at this https://www.youtube.com/watch?v=EumXak7TyQ0

Reply:
great. thanks for the advise 🤙